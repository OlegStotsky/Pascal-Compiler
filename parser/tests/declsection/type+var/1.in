type
a = float;

var
b : a;
c : a;


