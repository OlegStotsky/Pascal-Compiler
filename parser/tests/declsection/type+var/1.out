Syntax error at line 9, column 0 : BEGIN expected but EOF found
